extensions/wikia/Email/EmailController.class.php

```diff
@@ -62,7 +62,7 @@ public function init() {
 			$this->test = $this->getVal( 'test', false );
 			$this->marketingFooter = $this->request->getBool( 'marketingFooter' );
 
-			$noReplyName = wfMessage( 'emailext-no-reply-name' )->escaped();
+			$noReplyName = $this->getMessage( 'emailext-no-reply-name' )->escaped();
```

Comment:
This should probably have the ```inLanguage``` flag, so I don't think its an exception to my comment below.